### PR TITLE
feat: support perf client to show `TX` from client to server

### PR DESCRIPTION
**Pull request by jiuker**
## Motivation and Context

 show `TX` from client to server

## How to test this PR?

```shell
#> .\mc support perf client minio

Client: ✔

ENDPOINT                TX
http://127.0.0.1:9000   269 MiB/s
```

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Unit tests added/updated
- [ ] Internal documentation updated
- [ ] Create a documentation update request [here](https://github.com/minio/docs/issues/new?label=doc-change,title=Doc+Updated+Needed+For+PR+github.com%2fminio%2fmc%2fpull%2fNNNNN)

### CODE REVIEW

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: go.mod [hunk 1]
```diff
@@ -128,3 +128,5 @@ require (
 	google.golang.org/protobuf v1.31.0 // indirect
 	gopkg.in/ini.v1 v1.67.0 // indirect
 )
+
+replace github.com/minio/madmin-go/v3 => github.com/jiuker/madmin-go/v3 v3.0.0-20230807032717-2f88b58cc1fb
```
[the remark this answers is not in the document]
Comment (jiuker) [reply go.mod:132]: yes. I saw it. And I will make a change for 3.0.11. Hold on here

File: cmd/support-perf.go [hunk 2]
```diff
@@ -196,6 +197,14 @@ type NetTestResults struct {
 	Results []NetTestResult `json:"servers"`
 }
 
+// ClientResult - result of the network from client to server
+type ClientResult struct {
+	BytesSend uint64
```
Comment (shtripat) [inline cmd/support-perf.go:202]: %s/BytesSend/BytesSent/g

Comment (shtripat) [reply cmd/support-perf.go:202]: Also add `json` mapping for `BytestSent` and `TimeSpent`

**Review by shtripat — APPROVED** [review]
lgtm

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**